packages/drivers/routerlicious-driver/src/restWrapper.ts

```diff
@@ -155,7 +155,7 @@ export class RouterliciousOrdererRestWrapper extends RouterliciousRestWrapper {
 
     public static async load(
         tenantId: string,
-        documentId: string,
+        documentId: string | undefined,
```

Comment:
optional? or you don't want to change the order of the properties?

Reply:
trying to reduce the scope of the pr

Reply:
@alpaix @skylerjokiel code style question - is there a preference for this style vs `documentId?: string`? I'm wondering if we should standardize.

Reply:
You can declare an optional argument only when it comes after all mandatory ones. Here we have a few more arguments below. And re-ordering arguments would increase the pr's scope by introducing a breaking change.

Reply:
Thanks!